editor/project_manager.cpp

```diff
@@ -2688,6 +2688,10 @@ ProjectManager::ProjectManager() {
 		about_btn->set_text(TTR("About"));
 		about_btn->connect("pressed", callable_mp(this, &ProjectManager::_show_about));
 		tree_vb->add_child(about_btn);
+#ifdef ANDROID_ENABLED
+		// The About dialog is too large to display correctly on most Android devices.
+		about_btn->hide();
+#endif
```

Comment:
I think it would be a much better solution to allow shrinking the About dialog more. In `editor_about.cpp` line 164, I suggest changing the minimum size to (400, 200), at this size everything still fits nicely:

```cpp
	tc->set_custom_minimum_size(Size2(400, 200) * EDSCALE);
```

Reply:
I guess this can be removed now that #61215 is merged?